src/game_launcher_backend/game_deployer/deployer.mo

```diff
@@ -165,9 +165,93 @@ actor Deployer {
         Buffer.toArray(_b);
     };
 
+    private func _isController(collection_canister_id : Text, p : Principal) : async (Bool) {
```

Comment:
Please fix/revise naming conventions for args passed like (collection_canister_id -> game_canister_id).